### ISSUE TITLE
NodeJS server fails to handle serialization errors during a unary response

### ISSUE THREAD

**Issue by drobertduke** [grpc/grpc#9836]
The server's `sendUnaryResponse` does not call `handleError` correctly (it doesn't pass the `call` argument).  A server that tries to respond with an invalid message will produce this error instead of returning a status 13 to the client:
```
Uncaught TypeError: Cannot read property 'hasOwnProperty' of undefined
      at handleError (src/server.js:81:12)
      at sendUnaryResponse (src/server.js:135:5)
      at sendUnaryData (src/server.js:506:9)
      at Object.unary (test/server_test.js:120:11)
      at Object.wrapper [as func] (/Users/dduke/newtmp/projs/grpc/node_modules/lodash/lodash.js:4968:19)
      at src/server.js:499:13
```

Here's a test which reproduces the problem:
```
var grpc_client = require('../src/client.js');
var grpc = require('..');

var test_proto = grpc.load(__dirname + '/test_service.proto');
var test_service = test_proto.TestService.service;

describe('server', function() {
  it('handles serialization errors', function(done) {
    var test_server = new grpc.Server();
    test_server.addProtoService(test_service, {
      unary: function (call, callback) {
        call.sendMetadata(call.metadata);
        callback(null, {valid: false});
      }
    });
    var server_credentials = grpc.ServerCredentials.createInsecure();
    var test_port = test_server.bind('localhost:0', server_credentials);
    test_server.start();
    var TestClient = grpc_client.makeProtobufClientConstructor(test_service);
    var insecureCreds = grpc.credentials.createInsecure();
    var client = new TestClient('localhost:' + test_port, insecureCreds);
    client.unary(true, function(err) {
      assert.equal(err.message, '.Response#valid is not a field: undefined');
      test_server.forceShutdown();
      done();
    });
  });
});
```

**Comment by murgatroid99** [comment]
Thank you for reporting this, but it was actually previously reported in #9554, and fixed in #9456. The v1.1.x branch will eventually be merged into master, and those changes will appear in any future releases of the Node library.

**Comment by murgatroid99** [comment]
In fact, that fix should already be in the latest version of the Node gRPC library: 1.1.2.
